pyproject.toml

```
@@ -3,26 +3,28 @@ requires = ["setuptools>=64", "setuptools_scm>=8"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "napari_sphinx_theme"
+name = "napari-sphinx-theme"
 description = "Bootstrap-based Sphinx theme from the napari community"
 dynamic = ["version"]
 readme = "README.md"
 authors = [ {name = "napari team" }]
 
-requires-python = ">=3.7"
+requires-python = ">=3.13"
```

I think it's best to relax this for now, as we don't build napari docs with py313 yet.
Maybe we just match napari/napari python support?
```suggestion
requires-python = ">=3.10"
```

---

Sounds good to me. Thanks for the review.